ui/web-v2/src/components/FeatureAutoOpsRulesForm/index.tsx

```diff
@@ -894,10 +894,15 @@ const DateTimeOperation = memo(
       'YYYY/MM/DD HH:mm'
     );
 
-    const createdAt = dayjs(new Date(rule.createdAt * 1000)).format(
+    const updatedTime =
+      rule.createdAt === rule.updatedAt ? rule.createdAt : rule.updatedAt;
```

Comment:
`updatedTime` is always `rule.updatedAt`.

Reply:
@kentakozuka, fixed.